app/.env.docker-example

```diff
@@ -8,9 +8,9 @@ DJANGO_SUPERUSER_PASSWORD=admin
 
 # postgres settings for docker
 SQL_ENGINE=django.db.backends.postgresql
-SQL_DATABASE=people_depot_dev
-SQL_USER=people_depot
-SQL_PASSWORD=people_depot
+POSTGRES_DB=people_depot_dev
```

Comment:
This is right, but it's missing the lines from the other comment below. You should move the POSTGRES_ lines above the SQL_ lines to have a visual separation which keeps the POSTGRES_ lines together and the SQL_ lines together.